charts/internal/cloud-provider-config/values.yaml

```diff
@@ -1,4 +1,3 @@
-kubernetesVersion: 1.17.0
```

Comment:
I guess https://github.com/gardener/gardener-extension-provider-openstack/blob/024f99d4dc119437a999beac6c6763e4ce539c45/pkg/controller/controlplane/valuesprovider.go#L476 can be now removed.